apps/ui/src/components/forms/ContactForm.tsx

```diff
@@ -0,0 +1,108 @@
+"use client"
```

Comment:
I would move whole file to `components/elementary/forms`. `components/forms` should only contain form items like fields, select, checkbox etc. and not whole ready-to-go forms